sw/tests/helloworld.c

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Nicole Narr <narrn@student.ethz.ch>
+// Christopher Reinwardt <creinwar@student.ethz.ch>
+//
+// Simple payload to test bootmodes
+
+#include "regs/cheshire.h"
+#include "dif/clint.h"
+#include "dif/uart.h"
+#include "params.h"
+#include "util.h"
+
+int main(void) {
+    char str[] = "Hello World!\r\n";
+    uint32_t rtc_freq = *reg32(&__base_regs, CHESHIRE_RTC_FREQ_REG_OFFSET);
+    uint64_t reset_freq = clint_get_core_freq(rtc_freq, 2500);
+    uart_init(&__base_uart, reset_freq, 115200);
+    uart_write_str(&__base_uart, str, sizeof(str));
+    uart_write_flush(&__base_uart);
```

Comment:
Don't we support `printf`?

Reply:
Before the cleanup there was and I think also now, I just left the test as it was in cheshire